pkg/gce-pd-csi-driver/utils_windows.go

```diff
@@ -0,0 +1,71 @@
+// +build windows
+
+/*
+Copyright 2020 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gceGCEDriver
+
+import (
+	"fmt"
+	"k8s.io/utils/mount"
+	"sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/pkg/common"
+	mounter "sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/pkg/mount-manager"
+)
+
+func FormatAndMount(source, target, fstype string, options []string, m *mount.SafeFormatAndMount) error {
+	proxy, ok := m.Interface.(*mounter.CSIProxyMounter)
+	if !ok {
+		return fmt.Errorf("could not cast to csi proxy class")
+	}
+	return proxy.FormatAndMount(source, target, fstype, options)
+}
+
+func preparePublishPath(path string, m *mount.SafeFormatAndMount) error {
+	proxy, ok := m.Interface.(*mounter.CSIProxyMounter)
+	if !ok {
+		return fmt.Errorf("could not cast to csi proxy class")
+	}
+	exists, err := proxy.ExistsPath(path)
+	if err != nil {
+		return err
+	}
+	if exists {
+		return proxy.Unmount(path)
```

Comment:
Why do we need to unmount if it's already mounted?

Reply:
I added a comment. actually it is to remove the dir, not a real unmount like linux. I think could change the name if it is more clear.